core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -962,6 +962,8 @@ final class Stream[+F[_], +O] private[fs2] (private[fs2] val underlying: Pull[F,
       queue: Queue[F2, Option[Chunk[O2]]]
   ): Stream[F2, Nothing] = enqueueNoneTerminatedChunks(queue: QueueSink[F2, Option[Chunk[O2]]])
 
+  import Pull.StreamPullOps
```

Comment:
Why is the import necessary, is it not in implicit scope automatically?

Reply:
Failed to compile in Scala 3.